TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/ClawSlide.java

```diff
@@ -41,26 +41,26 @@ public ClawSlide(
                 leftSlide,
                 rightSlide);
         this.claw = new Claw(
-             clawRotLeft,
-             clawRotRight,
-             clawArmLeft,
-             clawArmRight);
+                clawRotLeft,
+                clawRotRight,
+                clawArmLeft,
+                clawArmRight);
```

Comment:
Can you change the indents to four spaces?
```suggestion
            clawRotLeft,
            clawRotRight,
            clawArmLeft,
            clawArmRight);
```